playbooks/roles/edxapp/templates/staticfiles_override.py.j2

```diff
@@ -1,3 +1,8 @@
 from .{{ edxapp_settings }} import *
 
+{% if item == 'cms' %}
+# cms uses a hashed directory to bust caches
+STATIC_ROOT = "{{ staticfiles_tmpdir.stdout }}" + "/" + EDX_PLATFORM_REVISION
```

Comment:
@OmarIthawi @melvinsoft sorry, I didn't pull in the context from my posting in slack yesterday here. `EDX_PLATFORM_REVISION` is what the `cms` `common.py` settings uses:

https://github.com/appsembler/edx-platform/blob/appsembler/amc/develop/cms/envs/common.py#L525

It is obtained via the [dealer](https://github.com/klen/dealer#basic-usage) library. The `git.revision` gets the first 7-digits of the git hash. (or it can be set via an environment variable, but I don't see us doing that in any of our branches).